oak_remote_attestation_verification/README.md

```diff
@@ -1,25 +1,12 @@
-# Oak Attestation Verification
-
-This crate provides functionality for verifying remote attestation evidence
-generated by Oak remote attestation protocol.
-
-## Verifying BinaryAttestation
-
-The `AttestationEndorsement` that an Oak client receives from the launcher
-contains a `BinaryAttestation` field that carries an endorsement statement
-generated by the Oak transparent release process and a Rekor log entry as the
-proof of inclusion of the endorsement statement in Rekor (the transparency log
-we use).
-
-Verifying the `BinaryAttestation` includes the following steps:
-
-1. Verifying the signature over the Rekor log entry (i.e., the log entry's
-   `signedEntryTimestamp` filed), using Rekor's public key to ensure that it is
-   indeed generated by Rekor.
-1. Verifying that the signature in the body of the Rekor log entry is over the
-   endorsement statement, and that the signature is produced by Oak. Doing this
-   requires having Oak's public key, or a certificate chain that proves the
-   public key belongs to Oak.
-1. Verifying that the SHA256 hash of the binary in the subject of the
-   endorsement statement must be the same as the hash of the remote server
-   binary (received in the remote attestation report).
+# Verification API
+
+Verification produces a go/no-go decision based on three (complex) arguments:
+
+1. **Evidence** is everything that is derived from the constellation, where a
```

Comment:
I would replace "constellation" here with "enclave". Anything that originates inside the enclave is considered evidence according to the RATS architecture (https://datatracker.ietf.org/doc/rfc9334/).

Reply:
Done.